### PR TITLE
Code generation for controllers without custom classes & other enhancements

### PR DESCRIPTION
**Pull request by Frizlab**
- On OSX, instead of the only method `instantiateControllerWithIdentifier()` which returns `AnyObject`, two methods are generated: `instantiateViewControllerWithIdentifier()` and `instantiateWindowControllerWithIdentifier()` returning respectively `NSViewController` and `NSWindowController`.
- Methods are generated for controllers which do not have a custom class.
- Segue return a destination even for destinations who don't have a custom class.
- Other small modifications (project upgrade with Xcode 7, indentation fixes, etc.)

**Note**: I did not test my modifications very much. Only tested on a small project on OSX.

### CODE REVIEW

**Comment by krzyzanowskim** [comment]
@Frizlab did you address this ticket: https://github.com/krzyzanowskim/Natalie/issues/36 ?

**Comment by Frizlab** [comment]
Partly, yes.
(Partly because segue for controllers which do not have a custom classes are still not accessible directly.)

File: natalie.swift [hunk 1]
```diff
@@ -969,7 +952,7 @@ class Storyboard: XMLObject {
                     print("")
                     print("//MARK: - \(customClass)")
 
-                    if let segues = scene.segues?.filter({ return $0.identifier != nil })
+                    if let segues = scene.segues?.filter({ return $0.identifier?.characters.count > 0 })
```
Comment (krzyzanowskim) [inline natalie.swift]: hm... if this is a case, should be handled upstream for "identifier"

Comment (Frizlab) [inline natalie.swift]: Will do that when I can (probably tomorrow).

Comment (Frizlab) [inline natalie.swift]: Pushed.

**Comment by krzyzanowskim** [comment]
thanks!

**Pull request merged**